ui/src/app/pages/profile/data-user-code-of-conduct.tsx

```diff
@@ -80,6 +80,19 @@ export const enum DuccSignatureState {
   SIGNED,
 }
 
+const VersionFailure = () => (
+  <>
+    <div>
+      Error: 'we are unable to display this version of the Data User Code of
```

Comment:
covers both cases: unsigned DUCC, or version is older than the first one we formatted for signed display (4).